<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings</class>
 <widget class="QDialog" name="settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>448</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imagesources.qrc">
    <normaloff>:/Images/settings.jpg</normaloff>:/Images/settings.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(143, 143, 143);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="9" column="4">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="6">
    <widget class="QCheckBox" name="m_chkboxChannel_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Arial Black&quot;;
color: rgb(236, 236, 236);</string>
     </property>
     <property name="text">
      <string>Channel 3</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QLabel" name="m_labelPacketSize">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Arial Black&quot;;
color: rgb(236, 236, 236);</string>
     </property>
     <property name="text">
      <string>Packet Size (KB)</string>
     </property>
    </widget>
   </item>
   <item row="13" column="6">
    <widget class="QCheckBox" name="m_chkboxChannel_4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Arial Black&quot;;
color: rgb(236, 236, 236);</string>
     </property>
     <property name="text">
      <string>Channel 4</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLineEdit" name="m_lineEditSaveFileName"/>
   </item>
   <item row="22" column="3">
    <widget class="QLabel" name="m_labelCOMPort">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Arial Black&quot;;
color: rgb(236, 236, 236);</string>
     </property>
     <property name="text">
      <string>COM Ports</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="m_labelADCChannel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Arial Black&quot;;
color: rgb(236, 236, 236);</string>
     </property>
     <property name="text">
      <string>ADC Channel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="m_lineEditSaveFileLocation">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </item>
   <item row="9" column="7">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>250</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="14" column="0" colspan="2">
    <widget class="QSpinBox" name="m_spinBoxReadLimit">
     <property name="minimum">
      <number>40</number>
     </property>
     <property name="maximum">
      <number>25600</number>
     </property>
     <property name="value">
      <number>400</number>
     </property>
    </widget>
   </item>
   <item row="13" column="3">
    <widget class="QLabel" name="m_labelNumberOfBuffer">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Arial Black&quot;;
color: rgb(236, 236, 236);</string>
     </property>
     <property name="text">
      <string>Number of buffer</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QSpinBox" name="m_spinBoxPacketSize">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(255, 255, 255);</string>
     </property>
     <property name="minimum">
      <number>4</number>
     </property>
     <property name="maximum">
      <number>256</number>
     </property>
     <property name="singleStep">
      <number>4</number>
     </property>
     <property name="value">
      <number>64</number>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="m_labelSaveFileName">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Arial Black&quot;;
color: rgb(236, 236, 236);</string>
     </property>
     <property name="text">
      <string>Save File Name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="m_labelSaveFileLoc">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Arial Black&quot;;
color: rgb(236, 236, 236);</string>
     </property>
     <property name="text">
      <string>Save File Location</string>
     </property>
    </widget>
   </item>
   <item row="24" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="13" column="5">
    <widget class="QCheckBox" name="m_chkboxChannel_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Arial Black&quot;;
color: rgb(236, 236, 236);</string>
     </property>
     <property name="text">
      <string>Channel 2</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QRadioButton" name="m_radioButtonFileType">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Arial Black&quot;;
color: rgb(236, 236, 236);</string>
     </property>
     <property name="text">
      <string>Text</string>
     </property>
    </widget>
   </item>
   <item row="14" column="3">
    <widget class="QLabel" name="m_labelReadLimit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Arial Black&quot;;
color: rgb(236, 236, 236);</string>
     </property>
     <property name="text">
      <string>Read Limit (MB)</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QSpinBox" name="m_spinBoxADCCahnnel"/>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="m_exploreButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="8">
    <widget class="QLineEdit" name="m_headerUSBSettings">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(44, 44, 44);
font: 87 8pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>USB Settings</string>
     </property>
    </widget>
   </item>
   <item row="9" column="5">
    <widget class="QCheckBox" name="m_chkboxChannel_1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Arial Black&quot;;
color: rgb(236, 236, 236);</string>
     </property>
     <property name="text">
      <string>Channel 1</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="2">
    <widget class="QSpinBox" name="m_spinBoxNumberOfBuffer">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>16</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="8">
    <widget class="QLineEdit" name="m_headerGeneralSettings">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(33, 33, 33);
font: 87 8pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>General Settings</string>
     </property>
    </widget>
   </item>
   <item row="21" column="0" colspan="8">
    <widget class="QLineEdit" name="m_headerUARTSettings">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(33, 33, 33);
font: 87 8pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>UART Settings</string>
     </property>
    </widget>
   </item>
   <item row="22" column="0">
    <widget class="QComboBox" name="m_comboBoxComPort">
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="23" column="3">
    <widget class="QLabel" name="m_labelBaudRate">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Arial Black&quot;;
color: rgb(236, 236, 236);</string>
     </property>
     <property name="text">
      <string>Baud Rate</string>
     </property>
    </widget>
   </item>
   <item row="23" column="0">
    <widget class="QComboBox" name="m_comboBoxBaudRate">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1152000</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="imagesources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
